src/app/branded/branded-checkout.component.js

```diff
@@ -17,19 +19,25 @@ let componentName = 'brandedCheckout';
 class BrandedCheckoutController {
 
   /* @ngInject */
-  constructor($window, analyticsFactory, tsysService) {
+  constructor($window, analyticsFactory, tsysService, sessionService) {
     this.$window = $window;
     this.analyticsFactory = analyticsFactory;
     this.tsysService = tsysService;
+    this.sessionService = sessionService;
   }
 
   $onInit() {
     this.code = this.designationNumber;
     this.tsysService.setDevice(this.tsysDevice);
-    this.checkoutStep = 'giftContactPayment';
-    this.formatDonorDetails();
     this.analyticsFactory.pageLoaded(true);
-    this.fireAnalyticsEvents('contact', 'payment');
+
+    this.sessionService.signOut().subscribe(() => {
+      this.sessionService.downgradeToGuest(true, false).subscribe(() => {
+        this.checkoutStep = 'giftContactPayment';
+        this.formatDonorDetails();
```

Comment:
Did this need to be moved to after the signout? It probably doesn't matter when it happens since it is pulling from bindings unless I'm missing something or the session change overwrites them.